TODO.md

```diff
@@ -21,6 +21,8 @@ Trying to reimplement the destructuring feature better (see #524 for a full intr
     - [x] lazy lists
     - [x] ranges
     - [x] When IR
+    - [x] Binary operation IR
```

Comment:
Make sure to remove this file 😉

Reply:
Yes, internal reminder... I'll remove and squash